src/ip_grid_mod.f90

```diff
@@ -73,11 +108,30 @@ end subroutine init_grib2_interface
 
 contains
 
+  !> Compares two grids.
+  !!
+  !! @param[in] grid1 An ip_grid
+  !! @param[in] grid2 Another ip_grid
+  !!
+  !! @return True if the grids are the same, false if not.
+  !!
+  !! @author Kyle Gerheiser
+  !! @date July 2021
   logical function is_same_grid(grid1, grid2)
     class(ip_grid), intent(in) :: grid1, grid2
     is_same_grid = grid1%descriptor == grid2%descriptor
   end function is_same_grid
-  
+
+  !> Returns the field position for a given grid point.
+  !!
+  !! @param[in] self
+  !! @param[in] i
+  !! @param[in] j
+  !!
+  !! @return Integer position in grib field to locate grid point.
+  !!
+  !! @author Kyle Gerheiser
+  !! @date July 2021
```

Comment:
You just wrote this code this month?

Do you have unit tests?

Reply:
I have updated the documentation with the correct authors and changed the date to its original 1996.